addons/account/views/account_move_views.xml

```diff
@@ -525,6 +525,13 @@
                            invisible="payment_state == 'invoicing_legacy' or state != 'posted' or move_type == 'entry'"
                            optional="show"/>
                     <field name="state" widget="badge" decoration-success="state == 'posted'" decoration-info="state == 'draft'" optional="show"/>
+                    <field name="move_sent_values"
+                            string="Sent"
+                            widget="badge"
+                            decoration-success="move_sent_values == 'Sent'"
+                            decoration-danger="move_sent_values == 'Not Sent'"
+                            column_invisible="context.get('default_move_type') not in ('out_invoice', 'out_refund', 'out_receipt')"
+                            optional="hide"/>
```

Comment:
since you put the parameter on one line let's do it all the way and putting the /> on the next line :smile: